dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewManagerScope.kt

```diff
@@ -49,10 +44,18 @@ internal class RumViewManagerScope(
 
     @WorkerThread
     override fun handleEvent(event: RumRawEvent, writer: DataWriter<Any>): RumScope {
+        if (!applicationDisplayed) {
+            val isForegroundProcess = CoreFeature.processImportance ==
```

Comment:
have you added a UnitTest for this case ? I didn't see it, maybe I missed it ?

Reply:
Yes, it's just they didn't change. They're `M start a foreground ViewScope W handleEvent(StartView)` in RumViewManagerScopeTest

Now it's with any event but I think other tests capture that.

Reply:
Yes it's just that they didn't change. They're `𝕄 start a foreground ViewScope 𝕎 handleEvent(StartView)` in `RumViewManagerScopeTest`. They only test for ViewEvents, but I think other test handle other possible events.